quality/integration/pom.xml

```diff
@@ -129,7 +129,6 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.17</version>
```

Comment:
Should we specify plugin version to the parent/pom.xml then?

Reply:
we get version from [bv-opensource-super-pom](https://github.com/bazaarvoice/emodb/blob/master/parent/pom.xml#L7)